pom.xml

```diff
@@ -61,6 +61,11 @@
 	</build>
 	<dependencies>
 		<dependency>
+			<groupId>org.apache.maven</groupId>
```

Comment:
I would like to avoid to add additional dependencies, do you think you can create a MR without this ?

Other than that i would like to merge it.